projects/15_basic_material/main.cpp

```diff
@@ -377,37 +393,92 @@ void ProjApp::SetupMaterials()
     createInfo.bindings.push_back({grfx::DescriptorBinding{CLAMPED_SAMPLER_REGISTER, grfx::DESCRIPTOR_TYPE_SAMPLER, 1, grfx::SHADER_STAGE_ALL_GRAPHICS}});
     PPX_CHECKED_CALL(GetDevice()->CreateDescriptorSetLayout(&createInfo, &mMaterialResourcesLayout));
 
+    // Green metal rust
+    {
+        SetupMaterialResources(
+            "poly_haven/textures/green_metal_rust/diffuse.png",
+            "poly_haven/textures/green_metal_rust/roughness.png",
+            "poly_haven/textures/green_metal_rust/metalness.png",
+            "poly_haven/textures/green_metal_rust/normal.png",
+            mWoodMaterial);
+        mMaterialResourcesSets.push_back(mWoodMaterial.set);
+    };
+
     // Wood
     {
         SetupMaterialResources(
-            "materials/textures/wood/albedo.png",
-            "materials/textures/wood/roughness.png",
-            "materials/textures/wood/metalness.png",
-            "materials/textures/wood/normal.png",
+            "poly_haven/textures/weathered_planks/diffuse.png",
+            "poly_haven/textures/weathered_planks/roughness.png",
+            "poly_haven/textures/weathered_planks/metalness.png",
+            "poly_haven/textures/weathered_planks/normal.png",
             mWoodMaterial);
         mMaterialResourcesSets.push_back(mWoodMaterial.set);
     };
 
     // Tiles
     {
         SetupMaterialResources(
-            "materials/textures/tiles/albedo.png",
-            "materials/textures/tiles/roughness.png",
-            "materials/textures/tiles/metalness.png",
-            "materials/textures/tiles/normal.png",
+            "poly_haven/textures/square_floor/diffuse.png",
```

Comment:
square_tiles ? or square_floor_tiles?

Reply:
Changed it to square_floor_tiles to make it very obvious what it is.